packages/forms/src/utils/withFieldSchema.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import * as React from 'react';
+
+import { FormContext } from '../FormContext';
+import type { FieldSchema, FormContextValue } from '../types';
+import { getFieldSchema } from './getFieldSchema';
+
+type FieldSchemaConsumerProps = {
+  fieldSchemaName?: string,
+};
+
+const withFieldSchema = () => (BaseComponent: React.ComponentType<*>) => {
```

Comment:
Why does `withFieldSchema` have two carried functions? First call is useless. There is no any value in closure.

Reply:
@vorobeez oops, misprint, nice catch 👍